### DISCUSSION TITLE
An option to only accept emails from already created senders/reverse-aliases or to the manually created ones on specified aliases

### DISCUSSION THREAD

**Discussion by scurvier** [simple-login/app#2088] [Feature Requests]
Hello,

I would like to prevent SPAM/Phishing by blocking all the emails that are delivered to my alias from new/unknown senders. I mean, I would like to have an option to have the block sender checkbox of the new automatically created reverse aliases checked by default in some aliases.

For example:
I could have an alias that I only use for my Bank account. All the legitimate reverse-aliases from all the email addresses of this Banking service are already created. I would like to block new senders (SPAM/Phishing emails) to deliver emails to me.
Currently, when I get a new SPAM, I have to go to SimpleLogin and block that reverse-alias. But as spammers usually use a new email address when they send a new SPAM campaign, I cannot really block them from delivery these emails to me. 

Regards

**Comment by kenny9p** [comment]
This is a great feature to implement. It's like having an option to turn on/off catch-all for an alias, just like how we can for a domain. 

It would be great if we're able to have a log of the blocked senders that didn't qualify to get forwarded, so we can check through and see if there are any new senders that we should allow through. What would be better is to have the ability to use regex to allow senders' email addresses that meet the criteria to have their emails get forwarded by the alias.
